### PR TITLE
Employees, invitations and approvers APIs

### PR DESCRIPTION
**Pull request by snarayanank2**
Introducing /admin/employees and /admin/employee_approvers paths

### CODE REVIEW

File: src/components/schemas/employee_approver.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+      $ref: './fields.yaml#/id_string'
+    org_id:
+      $ref: ./fields.yaml#/org_id
+    created_at:
+      $ref: ./fields.yaml#/created_at
+    updated_at:
+      $ref: ./fields.yaml#/updated_at
+    employee:
+      $ref: ./employee.yaml#/employee_out_embed
+    approver_ids:
```
Comment (jayachandhar) [inline src/components/schemas/employee_approver.yaml:15]: approver_ids should be employee ids

File: src/admin/paths/admin@employees.yaml [hunk 2]
```diff
@@ -6,6 +6,7 @@ get:
   parameters:
     - $ref: ../../components/parameters/created_at.yaml
     - $ref: ../../components/parameters/updated_at.yaml
+    - $ref: ../../components/parameters/is_enabled.yaml
```
Comment (aditya999123) [inline src/admin/paths/admin@employees.yaml:9]: what about status like pending_details ?

File: src/admin/paths/admin@employees.yaml [hunk 3]
```diff
@@ -34,31 +40,33 @@ get:
 post:
   tags:
     - Employees
-  summary: Create or update an employee
+  summary: update an employee
```
Comment (snarayanank2) [inline src/admin/paths/admin@employees.yaml:43]: Why remove `Create or`?

File: src/admin/paths/admin@invitations@bulk.yaml [hunk 4]
```diff
@@ -0,0 +1,40 @@
+post:
+  tags:
+    - Employees
+  summary: Create or update employees
+  description: |
+    Create or update an employee
+  operationId: employees_post_bulk
```
Comment (snarayanank2) [inline src/admin/paths/admin@invitations@bulk.yaml:7]: invitations_post_bulk

File: src/components/schemas/employee.yaml [hunk 5]
```diff
@@ -55,162 +52,193 @@ employee_out:
...
+    id:
+      $ref: './fields.yaml#/id_string'
+    user:
+      $ref: ./user.yaml#/user_out_embed
+    business_unit:
+      type: string
+      maxLength: 255
+      nullable: true
+      example: Finance ops
+    code:
```
Comment (snarayanank2) [inline src/components/schemas/employee.yaml:142]: remove business unit and include department_out_embed

File: src/components/schemas/employee.yaml [hunk 6]
```diff
@@ -55,162 +52,193 @@ employee_out:
...
 
-# TODO: need to fix this
 employee_in:
   type: object
+  additionalProperties: False
+  required: ["id"]
   properties:
     id:
       $ref: './fields.yaml#/id_string'
-    user_id:
```
Comment (snarayanank2) [inline src/components/schemas/employee.yaml:196]: you need user_id no? why removed it?

File: src/components/schemas/invitation.yaml [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+    per_diem_rate_names:
+      type: array
+      nullable: True
+      items:
+        $ref: './fields.yaml#/name'
+      example: [ 'TA - USD' ]
+      description: |
+        List of per diem rate names that this employee is allowed to access. If null, then there are no restrictions
+
+    vechicle_types:
```
Comment (snarayanank2) [inline src/components/schemas/invitation.yaml:88]: typo

File: src/admin/paths/admin@employees.yaml (hunk 2 above)
Comment (jayachandhar) [reply src/admin/paths/admin@employees.yaml:9]: we are deprecating pending_details in platform side

**Pull request merged**